packages/core/src/components/search/SearchInput/SearchInput.tsx

```diff
@@ -26,7 +26,8 @@ import {
 import type { SearchInputContextValue } from 'src/sdk/search/useSearchInput'
 import useOnClickOutside from 'src/sdk/ui/useOnClickOutside'
 
-import styles from './search-input.module.scss'
+import inputStyles from './search-input.module.scss'
+import styles from '../search.module.scss'
```

Comment:
css modules here caught my attention, because we are removing it on the other components, right?

Reply:
yep, but we didn't move all the `Search` feature yet 😬